src/devices/Bmxx80/Bmxx80Base.cs

```diff
@@ -135,24 +135,16 @@ public void Reset()
         }
 
         /// <summary>
-        /// Reads the temperature. A return value indicates whether the reading succeeded.
+        /// Reads the temperature from register.
         /// </summary>
-        /// <param name="temperature">
-        /// Contains the measured temperature if the <see cref="TemperatureSampling"/> was not set to <see cref="Sampling.Skipped"/>.
-        /// Contains <see cref="double.NaN"/> otherwise.
-        /// </param>
-        /// <returns><code>true</code> if measurement was not skipped, otherwise <code>false</code>.</returns>
-        public abstract bool TryReadTemperature(out Temperature temperature);
+        /// <returns>The measured <see cref="Temperature"/>.</returns>
+        protected abstract Temperature ReadTemperatureRegister();
 
         /// <summary>
-        /// Reads the pressure. A return value indicates whether the reading succeeded.
+        /// Reads the pressure from register.
         /// </summary>
-        /// <param name="pressure">
-        /// Contains the measured pressure in Pa if the <see cref="PressureSampling"/> was not set to <see cref="Sampling.Skipped"/>.
-        /// Contains <see cref="double.NaN"/> otherwise.
-        /// </param>
-        /// <returns><code>true</code> if measurement was not skipped, otherwise <code>false</code>.</returns>
-        public abstract bool TryReadPressure(out double pressure);
+        /// <returns>The measured pressure.</returns>
+        protected abstract double ReadPressureRegister();
```

Comment:
You should use the new Pressure unit here (refer to #768), same as it is with Temperature (property above it).

Reply:
Yeah I still need to rebase onto master, which I will do as soon as #783 is merged.

Reply:
But #768 is merged and is separate from #783.

Reply:
I see, will rebase onto current master then for now and use the new pressure unit of course.